browser/index.html

```diff
@@ -1,190 +1,240 @@
-<!-- https://github.com/jlebon/fedora-coreos-browser -->
+<!-- https://github.com/coreos/fedora-coreos-browser -->
 
 <!DOCTYPE html>
 <html lang="en">
     <head>
         <meta charset="utf-8">
-        <!-- PROD: <script src="https://cdn.jsdelivr.net/npm/vue"></script> -->
-        <!-- DEVEL: <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script> -->
-        <script src="https://cdn.jsdelivr.net/npm/vue"></script>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <!-- PROD: <script src="https://cdn.jsdelivr.net/npm/vue@2"></script> -->
```

Comment:
OOC, was the change for `@2` to fix something, or just because it's convention?

Reply:
I took it from https://vuejs.org/v2/guide/. It didn't work for me without that (the currently online version too).

Reply:
(though the latter may have been a temporary CDN glitch, it seems fine now).